documentation/projectGoals/ProjectGoals.tex

```diff
@@ -92,29 +92,29 @@ \section{Goals}
 \subsection{Low-Level Goals}
 The following items encompass the low-level goals of the system:
 \begin{itemize}
-\item The system can detect valid returns from the user with an accuracy of 90\%
-\item The system can hit each square of a 3x3 grid with an accuracy of 75\%
+\item The system can detect valid returns from the user with an accurately
+\item The system can hit 9 zones on the table, corresponding to a 3x3 grid
 \item The system implements reinforcement learning algorithms
-\item The system can provide insightful feedback to the user
+\item The system helps user improve their game in any way % made change to sound more like a goal
```

Comment:
Not big on the way this is worded. something more like "the system should help the user hone their skills"